main/src/cgeo/geocaching/log/LogTypeTrackable.java

```diff
@@ -1,18 +1,19 @@
 package cgeo.geocaching.log;
 
-import android.support.annotation.DrawableRes;
-import android.support.annotation.StringRes;
-
 import cgeo.geocaching.CgeoApplication;
 import cgeo.geocaching.R;
 import cgeo.geocaching.utils.Log;
+
+import android.support.annotation.DrawableRes;
 import android.support.annotation.NonNull;
+import android.support.annotation.StringRes;
+
 import java.util.ArrayList;
 import java.util.List;
 
 public enum LogTypeTrackable {
     DO_NOTHING(1, -1, "", R.string.log_tb_nothing, LogType.UNKNOWN),
-    VISITED(1001, 5, "_Visited", R.string.log_tb_visit, LogType.VISIT),
+    VISITED(1001, 5, "_Visited", R.string.log_tb_visit, LogType.VISIT, 75),
```

Comment:
Can we avoid the additional parameter by using LogType.VISIT.iconName instead?

Reply:
I think I know what you mean. You want to avoid the duplication of the number `75`. But somehow it doesn't feel right to use `LogType.VISIT.iconName` here. At this place it is not the name of an icon. Maybe the naming in the `LogType` class is also missleading.